src/transaction/log_tran_table.c

```diff
@@ -1457,6 +1458,28 @@ xlogtb_dump_trantable (THREAD_ENTRY * thread_p, FILE * out_fp)
   fprintf (out_fp, "\n");
 }
 
+/*
+ * logtb_get_tran_memory_info_nolatch - get the transaction memory information
+ *                                      with nolatch
+ *
+ * return: nothing
+ *
+ */
+void
+logtb_get_tran_memory_info_nolatch (std::vector < std::pair < int, uint64_t >> &tran_info)
+{
+  LOG_TDES *tdes;
+
+  for (int i = 0; i < NUM_TOTAL_TRAN_INDICES; i++)
+    {
+      tdes = log_Gl.trantable.all_tdes[i];
+      if (tdes != NULL && tdes->trid != NULL_TRANID && tdes->state == TRAN_ACTIVE)
+	{
+	  tran_info.push_back (std::make_pair (tdes->trid, tdes->cur_mem_usage.load ()));
```

Comment:
Nit. you can shoten this :).
```suggestion
	  tran_info.push_back ({tdes->trid, tdes->cur_mem_usage.load ()});
```
or
```suggestion
	  tran_info.emplace_back (tdes->trid, tdes->cur_mem_usage.load ());
```